app/models/patient.rb

```diff
@@ -633,6 +633,11 @@ def active_dependents_exclude_self
     active_dependents.where.not(id: id)
   end
 
+  # Get all dependents and always include self even if self is not active
+  def active_dependents_and_self
```

Comment:
How is this different than `active_dependents`?

Reply:
This is different in that it gets all dependents and **always** include self even if self is not active whereas if self is not active it will not be included in `active_dependents`